### ISSUE TITLE
error CS1056: Caractere '$'

### ISSUE THREAD

**Issue by JulioGold** [jhaker/nodejs-msbuild#5]
I'm receiving the message: "error CS1056: Caractere '$' inesperado" and the msbuild used come from "c:\windows\microsoft.net\framework\v4.0.30319\msbuild.exe", this path is created on "msbuild.prototype.getMSBuildPath" function of nodejs-msbuild lib, you sure this path is right?
Because I had an msbuild at "C:\Program Files (x86)\MSBuild\14.0\Bin\MSBuild.exe" and I believe wich this is right because this updated than another.
Anyway I'm put the "overrideParams.push('/tv:14.0');" and everything works ok.
Thank's.

**Comment by jhaker** [comment]
Thanks for the feedback. I am glad you were able to get past this with the overrides. Based on your feedback and some additional research I changed the default msbuild locations. I updated the project to paths for both "Program Files" and "Framework". I think this should now cover all visual studio versions up until now running x86, x64 or linux. 

These changes are included the latest version 0.3.0. Examples can be found on the updated README.md.
